release.sh

```diff
@@ -158,7 +158,7 @@ git push origin v${VERSION}
 git push origin ${REF_BRANCH}
 
 echo "==> Docker"
-docker build --no-cache --build-arg version=${VERSION} --tag gisaia/arlas-wui-builder:${VERSION} --tag gisaia/arlas-wui-builder:latest .
```

Comment:
I think there should be a similar mechanic as in the release script for ARLAS WUI where the latest tag is generated in certain situations, instead of fully removing it